dt/uni01alpha/kustomization.yaml

```diff
@@ -0,0 +1,202 @@
+        options:
+          create: true
+
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.cinderBackup.customServiceConfig
+    targets:
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.cinder.template.cinderBackup.customServiceConfig
+        options:
+          create: true
+
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.cinderBackup.replicas
+    targets:
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.cinder.template.cinderBackup.replicas
+        options:
+          create: true
+
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.glance.default.replicas
+    targets:
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.glance.template.glanceAPIs.default.replicas
+        options:
+          create: true
+
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.glance.customServiceConfig
+    targets:
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.glance.template.customServiceConfig
+        options:
+          create: true
+
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.glance.default.replicas
+    targets:
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.glance.template.glanceAPIs.default.replicas
+        options:
+          create: true
+
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.swift.enabled
+    targets:
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.swift.enabled
+        options:
+          create: true
+
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.ironic.enabled
+    targets:
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.ironic.enabled
+        options:
+          create: true
+
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.octavia.enabled
+    targets:
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.octavia.enabled
+        options:
+          create: true
+
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.telemetry.enabled
+    targets:
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.telemetry.enabled
+        options:
+          create: true
+
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.telemetry.autoscaling.enabled
+    targets:
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.telemetry.template.autoscaling.enabled
```

Comment:
if this get's enabled, it refers heatInstance: heat, so seems heat also needs to be enabled, without it it not completed. so something to check with telemetry

Reply:
Thanks @karelyatin will check with the telemetry team on it.

Reply:
Enabled `heat` but `telemetry` is still having issues.

Reply:
I still hit the issue and had to disable autoscaling and metricstorage to get it to Setup Complete

Reply:
Autoscaling requires several other resources including the MetricStorage and Heat. For MetricStorage to work, you need to have COO or OBO deployed in your cluster: https://github.com/openstack-k8s-operators/telemetry-operator/blob/079ca136696059ceb9e59a29091b1cdc2d89c69e/ci/create-coo-subscription.yaml#L12

Reply:
Thanks @vyzigold ,

```
[ciuser@titanamd126 ~]$ oc get operators -n openshift-operators
NAME                                                    AGE
barbican-operator.openstack-operators                   8h
cinder-operator.openstack-operators                     8h
cluster-observability-operator.openshift-operators      8h
designate-operator.openstack-operators                  8h
glance-operator.openstack-operators                     8h
heat-operator.openstack-operators                       8h
horizon-operator.openstack-operators                    8h
infra-operator.openstack-operators                      8h
ironic-operator.openstack-operators                     8h
keystone-operator.openstack-operators                   8h
kubernetes-nmstate-operator.openshift-nmstate           8h
manila-operator.openstack-operators                     8h
mariadb-operator.openstack-operators                    8h
metallb-operator.metallb-system                         8h
neutron-operator.openstack-operators                    8h
nova-operator.openstack-operators                       8h
observability-operator.openshift-operators              8h
octavia-operator.openstack-operators                    8h
openshift-cert-manager-operator.cert-manager-operator   8h
openstack-ansibleee-operator.openstack-operators        8h
openstack-baremetal-operator.openstack-operators        8h
openstack-operator.openstack-operators                  8h
ovn-operator.openstack-operators                        8h
placement-operator.openstack-operators                  8h
rabbitmq-cluster-operator.openstack-operators           8h
swift-operator.openstack-operators                      8h
telemetry-operator.openstack-operators                  8h
```

I believe we have enabled them.

Reply:
Issue is outside of the purview of this repo. All required changes required here are made.